### ISSUE TITLE
Property "queryStringPassThrough"

### ISSUE THREAD

**Issue by renankaic** [Axway-API-Management-Plus/apim-cli#517]
### APIM-CLI version

1.14.7

### API-Management version

7.7.0.20240830

### Bug description

I have the following content in one of my api-config file:
```json
  "inboundProfiles" : {
    "_default" : {
      "monitorAPI" : true,
      **"queryStringPassThrough" : true,**
      "monitorSubject" : "authentication.subject.id",
      "securityProfile" : "_default",
      "corsProfile" : "_default"
    }
  }
```
However, when importing back to the APIM, it seems that the queryStringPassThrough property isn't being set correctly:
![image](https://github.com/user-attachments/assets/51fb3dbf-a8a6-4d50-95f1-ab3a5024a6ff)

I can also see that on "PUT /api/portal/v1.4/proxies/{proxy-id}" requests, the "queryStringPassThrough" field isn't being sent:
![image](https://github.com/user-attachments/assets/3559d95a-5e21-4b1f-ac4f-178b06edbd88)

Please, is there anything else I should do to make this property to be set as "true" on API Gateway via apim cli?

Thank you in advance!

### Steps to reproduce

1. Export a frontend API
2. Open the api-config file and set the "queryStringPassThrough" as true
3. Import the frontend API again
4. Login into APIM and check if the "Query String Pass Through" is enabled (it'll be disabled)

**Comment by rathnapandi** [comment]
Hi @renankaic, 

Please test with 
[axway-apimcli-1.14.8-SNAPSHOT.tar.gz](https://github.com/user-attachments/files/17623848/axway-apimcli-1.14.8-SNAPSHOT.tar.gz)

**Comment by renankaic** [comment]
> Hi @renankaic,
> 
> Please test with [axway-apimcli-1.14.8-SNAPSHOT.tar.gz](https://github.com/user-attachments/files/17623848/axway-apimcli-1.14.8-SNAPSHOT.tar.gz)

It worked now! I tested with 3 frontend apis
![image](https://github.com/user-attachments/assets/a4d1f176-e84a-4630-92f4-a914d39388dd)

However, I noticed that when passing the flag "-o yaml", the backend swagger is being created as ".json" but with YAML format inside of it.

Please, when possible, could you fix this too? I mean, the file extension.

Thanks again for the support rathna!

**Comment by rathnapandi** [comment]
Thanks @renankaic for confirming. fixed the file extension issue, it will be part of next release.

**Comment by rathnapandi** [comment]
Fixed in 1.14.8
